components/EliminatedStatus/EliminatedStatus.tsx

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import * as React from 'react';
```

Comment:
Same here. Only export explicit dependencies being used.

Reply:
Updated in latest commit.